src/styles/blocks/header.scss

```
@@ -26,6 +26,11 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: color $transition-time ease;
+
+    &:hover {
```

still no effect( if you have any troubles ask us some help in the chat please. I think you should change color of `a` element (nav__link) instead of `nav __item`. If it will works - then you would better move `transition` propery to link selector 
```suggestion
    &:hover > .nav__link  {
```